zulipterminal/ui_tools/boxes.py

```diff
@@ -725,14 +727,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     )
                     if not all_valid:
                         return key
-                    # We extract emails into self.recipient_emails only once we know
+                    # We extract recipients' user_ids and emails only once we know
                     # that all the recipients are valid, to avoid including any
                     # invalid ones.
-                    self.update_recipient_emails(self.to_write_box)
-                    users = self.model.user_dict
-                    self.recipient_user_ids = [
-                        users[email]["user_id"] for email in self.recipient_emails
-                    ]
```

Comment:
This is part of the `update_recipients` method now, so removing these lines as well.